### PR TITLE
`chain-spec-builder`: info about patch/full files added

### PR DESCRIPTION
**Pull request by michalkucharczyk**
There was no good example of what is patch and full genesis config file. Some explanation and example were added to the `chain-spec-builder` doc.

### CODE REVIEW

File: substrate/bin/utils/chain-spec-builder/README.docify.md [hunk 1]
```diff
@@ -84,17 +86,59 @@ Build the chain spec using provided full genesis config json file. No defaults w
...
+
 ### Generate human readable chain spec using provided full genesis config
 
 <!-- docify::embed!("tests/test.rs", cmd_create_full_plain)-->
 
+Refer to [*full config file*](#full-genesis-config-file) for some details on the full file format.
+
+
+## Patch and full genesis config files
+This section provides details on the files that cane be used with `create patch` or `create full` subcommands.
```
Comment (iulianbarbu) [inline substrate/bin/utils/chain-spec-builder/README.docify.md:108]:
```suggestion
This section provides details on the files that can be used with `create patch` or `create full` subcommands.
```

File: substrate/bin/utils/chain-spec-builder/README.docify.md [hunk 2]
```diff
@@ -84,17 +86,59 @@ Build the chain spec using provided full genesis config json file. No defaults w
...
+	},
+	"sudo": {
+		"key": "5Ff3iXP75ruzroPWRP2FYBHWnmGGBSb63857BgnzCoXNxfPo"
+	}
+}
+```
+The rest of genesis config keys will be initialized with default values.
+
+### Full genesis config file.
+The full genesis config file shall contain values for *all* the keys present in the genesis config for given runtime. The format of the file is
```
Comment (iulianbarbu) [inline substrate/bin/utils/chain-spec-builder/README.docify.md:139]:
```suggestion
The full genesis config file must contain values for *all* the keys present in the genesis config for given runtime. The format of the file is
```

**Comment by michalkucharczyk** [comment]
/cmd prdoc --force  --bump patch --audience runtime_user

File: prdoc/pr_6373.prdoc [hunk 3]
```diff
@@ -0,0 +1,8 @@
+title: '`chain-spec-builder`: info about patch/full files added'
+doc:
+- audience: Runtime User
+  description: There was no good example of what is patch and full genesis config
+    file. Some explanation and example were to the `chain-spec-builder` doc.
```
Comment (michalkucharczyk) [inline prdoc/pr_6373.prdoc:5]:
```suggestion
    file. Some explanation and example were added to the `chain-spec-builder` doc.
```

**Review by iulianbarbu — APPROVED** [review]

File: substrate/bin/utils/chain-spec-builder/README.docify.md [hunk 4]
```diff
@@ -84,17 +86,59 @@ Build the chain spec using provided full genesis config json file. No defaults w
 
 <!-- docify::embed!("tests/test.rs", cmd_create_full_raw)-->
 
+Refer to [*full config file*](#full-genesis-config-files) for some details on the full file format.
```
Comment (iulianbarbu) [inline substrate/bin/utils/chain-spec-builder/README.docify.md:89]:
```suggestion
Refer to [*full config file*](#full-genesis-config-file) for some details on the full file format.
```

**Review by kianenigma — APPROVED** [review]

**Pull request merged**